### PR TITLE
OCPBUGS-44920: Remove trailing dot from the i18n error msg

### PR DESCRIPTION
**Pull request by jhadvig**
/assign @rhamilto

### CODE REVIEW

File: frontend/public/i18n.js [hunk 1]
```diff
@@ -109,7 +109,7 @@ export const init = () => {
       },
       saveMissing: true,
       missingKeyHandler: function (lng, ns, key) {
-        window.windowError = `Missing i18n key "${key}" in namespace "${ns}" and language "${lng}."`;
+        window.windowError = `Missing i18n key "${key}" in namespace "${ns}" and language "${lng}"`;
```
Comment (rhamilto) [inline frontend/public/i18n.js:112]:
```suggestion
        window.windowError = `Missing i18n key "${key}" in namespace "${ns}" and language "${lng}".`;
```

Comment (rhamilto) [reply frontend/public/i18n.js:112]: I'm thinking we still want the period, but outside the quotes.

**Comment by jhadvig** [comment]
/label tide/merge-method-squash

@rhamilto comment addressed 👍

**Review by rhamilto — APPROVED** [review]
/lgtm

**Comment by logonoff** [comment]
/retitle OCPBUGS-44920: Remove trailing dot from the i18n error msg

**Pull request merged**